lib/Github/Api/Repo.php

```diff
@@ -535,6 +535,25 @@ public function branches($username, $repository, $branch = null, array $paramete
         return $this->get($url, $parameters);
     }
 
+    /**
+     * Sync a fork branch with the upstream repository.
+     *
+     * @link https://docs.github.com/en/rest/branches/branches#sync-a-fork-branch-with-the-upstream-repository
+     *
+     * @param string      $username
+     * @param string      $repository
+     * @param string|null $branchName
```

Comment:
```diff
diff --git a/lib/Github/Api/Repo.php b/lib/Github/Api/Repo.php
index 765c840..7536d70 100644
--- a/lib/Github/Api/Repo.php
+++ b/lib/Github/Api/Repo.php
@@ -540,13 +540,9 @@ class Repo extends AbstractApi
      *
      * @link https://docs.github.com/en/rest/branches/branches#sync-a-fork-branch-with-the-upstream-repository
      *
-     * @param string      $username
-     * @param string      $repository
-     * @param string|null $branchName
-     *
      * @return array|string
      */
-    public function mergeUpstream($username, $repository, $branchName)
+    public function mergeUpstream(string $username, string $repository, string $branchName)
     {
         return $this->post(
             '/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/merge-upstream',
```

One last change I missed in the previous review, otherwise the PR is good to go!

Reply:
the method is written like the rest of the code, if typing is added throughout the code, I will add it to the method.